toolkit/tools/internal/file/file.go

```diff
@@ -235,6 +236,21 @@ func Append(data string, dst string) (err error) {
 	return
 }
 
+// AppendLines appends multiple lines to the end of a file.
+func AppendLines(lines []string, dst string) error {
+	for _, line := range lines {
+		// Check if the line already ends with a newline character.
+		if !strings.HasSuffix(line, "\n") {
+			// Add newline if missing.
+			line += "\n"
+		}
+		if err := Append(line, dst); err != nil {
```

Comment:
`Append` opens and closes the file. So, this function is very inefficient. You should ensure the file is only opened once.